### ISSUE TITLE
Is it possible to forget a password in a given website?

### ISSUE THREAD

**Issue by JPUnD** [browserpass/browserpass-extension#259]
Maybe this is already supported, but I can't figure out how to make it forget a password that was used in a given website by accident. 
Any pointers?

**Comment by maximbaz** [comment]
There is no UI for this yet, although I think it's something we wanted to eventually add to the context menu.

There is "Clear usage data" in extension options if you are okay with deleting _all_ usage data... 

Technically it is possible to delete a single entry (we store it in localstorage), but it's stored in a [hashed format](https://github.com/browserpass/browserpass-extension/blob/77f6d18f8ea92e68fc0c020cb4065283e45a78c2/src/background.js#L226), so it will be a bit tricky to reverse engineer to find the correct entry... But if you want to try, either put a breakpoint there and login again with the incorrect entry (to see it's hash), or calculate it by hand: if you didn't configure a password store in the settings, its name is "default", origin is the first part of the URL, e.g. " https://github.com" on github, and entry is what you see in browserpass popup, eg. "personal/github.com".

Local storage of the extension can be accessed e.g. in Chromium via settings > extensions > Browserpass > background page > Application tab > local storage, and look there for "recents" key.

Hope that helps for now 🙂

**Comment by JPUnD** [comment]
I'll have to work with it, but maybe it's easier to delete the file and just create it again.

I think this could be a good feature request, should i be submitting it somewhere? (that's all i can do with my zero knowledge on java script)

**Comment by maximbaz** [comment]
Yeah just hitting the "Clear usage data" might be the simplest way to go.

This ticket will serve well as a feature request 👍
